From 8abc251dc0d801fae2ad8dc6c27f1509788179b6 Mon Sep 17 00:00:00 2001
From: Chen Baozi <chenbaozi@phytium.com.cn>
Date: Mon, 17 Jun 2024 19:33:00 +0800
Subject: [PATCH 009/150] dt-bindings: arm: Document Phytium FTC series cpu

Document the compatible string for the FTC series cpus found in Phytium
SoCs.

Signed-off-by: Chen Baozi <chenbaozi@phytium.com.cn>
Change-Id: I0c64b28ef3c89873fadafa8c71c57226e43bcecf
Signed-off-by: Andrew Powers-Holmes <aholmes@omnom.net>
---
 Documentation/devicetree/bindings/arm/cpus.yaml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/cpus.yaml b/Documentation/devicetree/bindings/arm/cpus.yaml
index 0d17cbad9778..c699bf4f33f9 100644
--- a/Documentation/devicetree/bindings/arm/cpus.yaml
+++ b/Documentation/devicetree/bindings/arm/cpus.yaml
@@ -182,6 +182,12 @@ properties:
       - nvidia,tegra132-denver
       - nvidia,tegra186-denver
       - nvidia,tegra194-carmel
+      - phytium,ftc310
+      - phytium,ftc660
+      - phytium,ftc661
+      - phytium,ftc662
+      - phytium,ftc663
+      - phytium,ftc664
       - qcom,krait
       - qcom,kryo
       - qcom,kryo240
-- 
2.47.0

